types/string/fixed_length.ts

```diff
@@ -0,0 +1,25 @@
+import { SizedType } from "../types.ts";
+
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+
+export class FixedLengthString implements SizedType<string> {
+  byteLength: number;
```

Comment:
should this be readonly?

Reply:
Maybe makes sense, but I don't really like readonly as it is only a typescript type annotation like private, public and protected. No one stops you from js to actually change it unless you are using a getter and a `#private` hidden value.

https://github.com/denosaurs/byte_type/blob/717f3156b37e028171783c684964dd484714ccae/types/types.ts#L7-L9